Dockerfile.bm-inventory-build

```
@@ -1,15 +1,35 @@
-FROM golang:1.14.3
+FROM centos:7
```

My contention would be that ubi (and ubi8 for that matter) would be the best choice to use upstream as it would most closely resemble d/s builds.

```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

---

But aren't you able to override the `FROM` for the purposes of prow builds?